test/test_linalg.py

```diff
@@ -7290,7 +7293,8 @@ def test_slogdet_errors_and_warnings(self, device, dtype):
             with self.assertRaisesRegex(RuntimeError, "tensors to be on the same device"):
                 torch.linalg.slogdet(a, out=(sign_out, logabsdet_out))
 
-    @skipCUDAIf(torch.version.cuda.split(".") < ("11", "3"), "There's a bug in cuSOLVER < 11.3")
+    @skipCUDAIf(torch.version.cuda is not None
+                and torch.version.cuda.split(".") < ("11", "3"), "There's a bug in cuSOLVER < 11.3")
```

Comment:
There is also `torch.testing._internal.common_cuda._get_torch_cuda_version()`.

Reply:
Funnily enough, while I cannot use that function because it returns (0, 0) when there's no cuda and I don't want to skip this in that case. I could use that odd operator of `a < x < b` from Python, but I don't really like it.
Good to know it exists though.